types/react-responsive-spritesheet/index.d.ts

```diff
@@ -0,0 +1,71 @@
+// Type definitions for react-responsive-spritesheet 1.0
+// Project: https://github.com/danilosetra/react-responsive-spritesheet
+// Definitions by: Michael Liu <https://github.com/triangularcube>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+import * as CSS from 'csstype';
+
+declare class ReactResponsiveSpritesheet extends React.Component<ReactResponsiveSpritesheet.Props> {
```

Comment:
you can export this directly:
```suggestion
export default class ReactResponsiveSpritesheet extends React.Component<Props> {
```

Reply:
Thanks a lot, I've updated the PR.